File: GbrainsJavaProject/app/src/main/java/com/nbgsoftware/gbrains/customview/editText/CustomEditText.java
package com.nbgsoftware.gbrains.customview.editText;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.constraintlayout.widget.ConstraintLayout;

import com.nbgsoftware.gbrains.databinding.LayoutCustomEditTextBinding;

public class CustomEditText extends ConstraintLayout {
    private LayoutCustomEditTextBinding binding;

    public CustomEditText(@NonNull Context context) {
        super(context);
    }

    public CustomEditText(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public CustomEditText(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public CustomEditText(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
        super(context, attrs, defStyleAttr, defStyleRes);
    }
}

File: GbrainsKotlinProject/app/src/main/java/com/nbgsoftware/gbrains/custom/CustomEditText.kt
package com.nbgsoftware.gbrains.custom

import android.content.Context
import android.util.AttributeSet
import androidx.constraintlayout.widget.ConstraintLayout
import com.nbgsoftware.gbrains.databinding.CustomViewEditTextBinding

class CustomEditText : ConstraintLayout {

    private lateinit var binding: CustomViewEditTextBinding

    constructor(context: Context) : super(context)
    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    )
}